pkg/filters/args.go

```diff
@@ -106,7 +106,7 @@ func (filter *ArgFilter) Parse(filterName string, operatorAndValues string, even
 	}
 
 	// if the event is a signature event, we allow filtering on dynamic argument
-	if !argFound && !eventDefinition.IsASignatureEvent() {
+	if !argFound && !events.IsASignatureEvent(eventDefinition.ID32Bit) {
```

Comment:
Are you sure we need to use ID32Bit and not ID? ID32Bit is intended to map 64 bit syscalls to their matching 32 bit compat syscalls

Reply:
@yanivagman maybe, but what I see is -> 
[`Event`](https://github.com/aquasecurity/tracee/blob/main/pkg/events/events.go#L177C1-L177C1) which is what we get when we do an event definition [`Get`](https://github.com/aquasecurity/tracee/blob/main/pkg/events/events.go#L239) or [`GetSafe`](https://github.com/aquasecurity/tracee/blob/main/pkg/events/events.go#L245), and Event only has [`ID32Bit`](https://github.com/aquasecurity/tracee/blob/main/pkg/events/events.go#L178), which has type `ID`, the field ID itself only exists for [`eventDependency`](https://github.com/aquasecurity/tracee/blob/main/pkg/events/events.go#L170), but both have the same type. 

When we define an `Event` for the event definitions table, I only see we setting `ID32Bit` -> https://github.com/aquasecurity/tracee/blob/main/pkg/events/events.go#L509

Isn't it correct than?

Reply:
The key of the events map in the eventsDefinitions struct is also of type ID.
You can simply use the id local variable that is already defined above in this function

Reply:
sure

Reply:
Related: https://github.com/aquasecurity/tracee/pull/3293